### PR TITLE
NOISSUE - Enhance opcua docs

### PR DESCRIPTION
**Pull request by WashingtonKK**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Enhances the OPC-UA docs and adds an illustration of how it can be used.

### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)
None

### CODE REVIEW

**Review by dborovcanin — COMMENTED** [review]
@WashingtonKK Can you align the look and feel of the diagram with other diagrams in the docs?

File: docs/opcua.md [hunk 1]
```diff
@@ -58,7 +61,16 @@ To create an OPC-UA subscription, user should connect the Thing to the Channel.
 
 To forward OPC-UA messages the opcua-adapter subscribes to the Node ID of an OPC-UA Server URI. It verifies the `server_uri` and the `node_id` of received messages. If the mapping exists it uses corresponding `Channel ID` and `Thing ID` to sign and forwards the content of the OPC-UA message to the Mainflux message broker. If the mapping or the connection between the Thing and the Channel don't exist the subscription stops.
 
+### Sample Use Case
+
+OPC-UA can be used in an industrial setup to monitor process values from the different industrial devices and machines within the industry setup. The industrial devices which are controlled by controllers such as PLCs (Programmable Logic Controllers) send data to the OPC-UA sever over TCP/IP. From the OPC-UA server, data is sent to and from Mainflux cloud using the OPC-UA adapter in Mainflux. The devices, known as nodes are associated with things on Mainflux and the Service URIs are associated with Channels on Mainflux. Data (Messages) received on the channels can be sent to the writers on Mainflux to make it available to the end user.
+
+|       ![OPC-UA][opcua-diagram]        |
```
Comment (drasko) [inline docs/opcua.md]: PLease align this nicely, killing the extra space on the right side

**Review by drasko — CHANGES_REQUESTED** [review]

**Comment by drasko** [comment]
@WashingtonKK please fix this diagram so that we can move on. @SammyOina please review this PR.

**Comment by WashingtonKK** [comment]
@drasko , I had updated the image. Now I'm waiting for remarks on the image and the PR

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**